modules/cluster-bare-metal-operator.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// *  operators/operator-reference.adoc
+// *  post_installation_configuration/cluster-capabilities.adoc
+
+// operators/operator-reference.adoc
+ifeval::["{context}" == "cluster-operators-ref"]
+:operator-ref:
+endif::[]
+
+// post_installation_configuration/cluster-capabilities.adoc
+ifeval::["{context}" == "cluster-capabilities"]
+:cluster-caps:
+endif::[]
+
+:_content-type: REFERENCE
+[id="cluster-bare-metal-operator_{context}"]
+ifdef::operator-ref[= Bare Metal Operator]
+ifdef::cluster-caps[= Bare-metal capability]
+
+ifdef::operator-ref[]
+
+[NOTE]
+====
+The Bare Metal Operator is an optional cluster capability that can be disabled by cluster administrators during installation. For more information about optional cluster capabilities, see "Cluster capabilities" in _Post-installation configuration_.
+====
+
+endif::operator-ref[]
+
+[discrete]
+== Purpose
+
+ifdef::cluster-caps[]
+
+The Bare Metal Operator provides the features for the `baremetal` capability.
+
+endif::cluster-caps[]
+
+The Bare Metal Operator (BMO) enables management of bare-metal hosts to be used as nodes in the cluster. When deploying OpenShift Container Platform on bare-metal hosts, you might need to make changes to the host either before or after provisioning.
+
+The Bare Metal Operator is only necessary when running clusters with bare-metal hosts.
```

Comment:
Should we document the implications of disabling baremetal capability, especially in bare-metal cluster?

Reply:
@shellyyang1989 I would happily add that content. Do you know of a resource that documents the impact of disabling the baremetal capability in a bare-metal cluster? Or a good place to find more information on the topic?

Reply:
Thanks Michael! @pamoedom could you please help with ^? Thanks!

Reply:
cc @elfosardo to help with this question about the impacts of not enabling BMO

Reply:
you could add that disabling BMO would mean not being able to interact with the BM cluster at all
the BMO is the component that talks to ironic, without that there's no way to manage any bare metal host
in general, in the case of an IPI deployment, disabling BMO does not make sense

Reply:
Thank you for the feedback. Please let me know what you think of the changes.